earthpy/spatial.py

```diff
@@ -61,12 +64,12 @@ def normalized_diff(b1, b2):
         raise ValueError("Both arrays should be of the same dimensions")
 
     n_diff = (b2 - b1) / (b2 + b1)
-    #ndvi[np.isnan(ndvi)] = 0
     n_diff = np.ma.masked_invalid(n_diff)
     return n_diff
 
 
-# TODO: include a no data value here if provided ...
+# TODO: include a no data value here if provided 
+# TODO: Include .sort() in the function
```

Comment:
this is another example of making a suggestion that the developers should approve. In this case i would argue that sort should happen after glob and before being passed to this function. This function's intent is not to order lists but rather to create a stack. It assumes the user passes what it wants. Please remove this  line frm your PR. 

A todo should be a conversation with the developers that they
1. agree to do and then
2. ideally your pr would fix that issue. 
Here i disagree that this is a todo.